pkg/apis/apps/v1alpha1/types.go

```diff
@@ -94,9 +93,9 @@ type ApplicationStatus struct {
 
 // applicationSourceStatus defines the observed state of the artifact source.
 type ApplicationSourceStatus struct {
-	GitRepoStatus  *sourcev1.GitRepositoryStatus    `json:"gitRepoStatus,omitempty"`
```

Comment:
Why cannot we use V1.GitRepo

Reply:
v1 is the latest version. we are currently using v1beta2.

Reply:
we're using [helm](https://github.com/fluxcd-community/helm-charts/releases/tag/flux2-2.7.0) to install flux, currently the helm chart does not support v1

Reply:
we need wait for next release with this PR: https://github.com/fluxcd-community/helm-charts/pull/176